webapp/components/user_settings/import_theme_modal.jsx

```diff
@@ -84,7 +84,7 @@ class ImportThemeModal extends React.Component {
         const user = UserStore.getCurrentUser();
         user.theme_props = theme;
 
-        Client.updateUser(user,
+        Client.updateUser(user, 'theme',
```

Comment:
Minor but we should probably use constants for those strings, 'theme, 'language', etc.